index.html

```diff
@@ -1,111 +1,424 @@
+        <button>Read more</button>
+      </section>
+      <section class="our-team">
+        <div>
+          <h1>Our team</h1>
+        </div>
+        <div>
+          <div class="avatar">
+            <img src="#" alt="text" />
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h1>Jason Smith</h1>
+            <p>Co-Founder</p>
+            <span class="icon">Twitter</span>
+            <span class="icon">Facebook</span>
+            <span class="icon">Telegram</span>
+            <span class="icon">LinkedIn</span>
+            <span class="icon">Instagram</span>
+          </div>
+        </div>
+      </section>
+      <section class="faqs">
+        <div>
+          <h1>FAQs</h1>
+        </div>
+        <div>
+          <div>
```

Comment:
This block could also be made a 'details' tag. Then next tag should be 'summary'. And '+' element you can add in styles

Reply:
done